### DISCUSSION TITLE
Protontricks "invalid file magic number"

### DISCUSSION THREAD

**Discussion by xlomannix** [Matoking/protontricks#371] [Q&A]
Linux newbie here, so sorry in advance if this has an obvious fix.

I'm trying to mod Skyrim SE via this guide ( https://www.nexusmods.com/skyrimspecialedition/mods/91500?tab=description ) and I'm on step 4. Problem is, whenever I type `protontricks --gui --verbose` I get this result: 
```
protontricks (INFO): Found Steam directory at /home/lomanni/.local/share/Steam
protontricks (INFO): Using default Steam Runtime at /home/lomanni/.local/share/Steam/ubuntu12_32/steam-runtime
protontricks (INFO): Winetricks path is set to /usr/bin/winetricks
protontricks (INFO): Found 1 Steam library folders
protontricks (INFO): Currently logged-in Steam user: 1loma
protontricks (INFO): Couldn't find custom shortcuts. Maybe none have been created yet?
protontricks (INFO): Using 'zenity' as GUI provider
Traceback (most recent call last):
  File "/usr/bin/protontricks", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/usr/lib/python3/dist-packages/protontricks/cli/main.py", line 32, in cli
    main(args)
  File "/usr/lib/python3/dist-packages/protontricks/cli/util.py", line 149, in wrapper
    return cli_func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/protontricks/cli/main.py", line 274, in main
    proton_app = find_proton_app(
                 ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/protontricks/steam.py", line 821, in find_proton_app
    tool_app = find_steam_compat_tool_app(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/protontricks/steam.py", line 610, in find_steam_compat_tool_app
    appinfo_sections = [
                       ^
  File "/usr/lib/python3/dist-packages/protontricks/steam.py", line 540, in iter_appinfo_sections
    raise SyntaxError("Invalid file magic number")
SyntaxError: Invalid file magic number
```
I've tried repeating previous steps, like:
```
export WINE="/usr/bin/wine"
export WINETRICKS="/usr/bin/winetricks"
echo $WINE
echo $WINETRICKS
/usr/bin/wine
/usr/bin/winetricks
```
And I've done some troubleshooting with ChatGPT, but nothing has worked so far. Any help would be greatly appreciated.

**Comment by MatthewForrester** [comment]
I am no expert, but I had the same problem and I can tell you how I fixed it.

Based on bug reports over on the Issue tab, it seems that Steam changed something in recent versions which breaks older versions of Protontricks. Sou need at least version 1.12.0 of Protontricks. If you installed Protontricks from your distribution, then it's probably too old (e.g. even the latest release of Ubuntu, 24.10 Oracular, only has 1.11.1).

You can get [the latest version of Protontricks](https://flathub.org/apps/com.github.Matoking.protontricks) from Flathub. If you've never heard of Flathub or Flatpaks, [start here](https://docs.flathub.org/) so you know what you are doing.
